doc/source/index.rst

```diff
@@ -40,17 +40,202 @@ An example of such a callable class used in trollflow2 is the
 `class:FilePublisher`.
 
 
-Available plugins
-+++++++++++++++++
+Main plugins
+++++++++++++
 
 .. currentmodule:: trollflow2.plugins
 
-The `check_sunlight_coverage` plugin
-************************************
+Scene creation
+**************
+
+The `create_scene` plugin creates a scene object.  The configuration
+options are:
```

Comment:
Should we make it clear that the configuration options are to be specified in the product list, and not in the `workers` section ?

Reply:
Yes, good point.